src/style.css

```
@@ -1,3 +1,67 @@
+html {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
+  justify-content: space-between;
+  text-wrap: nowrap;
+  box-sizing: border-box;
+}
+
+header {
```

Use only class selector
```suggestion
.header {
```